environments/sandbox.yml

```diff
@@ -47,7 +47,12 @@ A:
     ttl: 60
     record:
       - "51.104.255.30"
```

Comment:
please change this to 20.49.142.14, its changed because I have re-deployed it